src/katsdpk8spoc/sdpcontroller.py

```diff
@@ -0,0 +1,368 @@
+################################################################################
```

Comment:
I did something wrong with the rename of the file. It now looks like a new file.

Reply:
It's a bit harder to try and do a diff, but otherwise no worries.